version.json

```diff
@@ -1,4 +1,4 @@
 {
-    "adapt_authoring": "v0.2.2",
-    "adapt_framework": "v2.0.7"
+    "adapt_authoring": "v0.3.1",
+    "adapt_framework": "v2.0.16"
```

Comment:
that's true
plus can you add me to the contributors please https://github.com/adaptlearning/adapt_framework/blob/master/package.json#L115-L116